+USI/Sounding Rockets

minchinweb opened the pull request:
The only thing I wasn't sure was what to name the `SR_InlineProbe`. It is 0.3m in diameter, so "RC-03" seemed like the most logical (to match "RC-12"), but the -01, -02, and -03 are used (for the stock parts) to refer to the probe level rather than diameter...

Tested in-game with KSP 1.12.3 and Sounding Rockets 1.4.0.

minchinweb committed:
+Sounding Rockets

yalov commented:
I would put it in the end of the all RC-03s,  something like 
"RC-03 'Umbra' Probe Core M-315"

yalov commented:
Are you using only the Sounding Rockets from the USI kit, or the other USIs will follow ?

minchinweb committed:
per review updates

minchinweb commented:
I've changed these patches to be single line like you requested for the other PR.
I've also updated the name of the probe core.

> Are you using only the Sounding Rockets from the USI kit, or the other USIs will follow ?

Immediately, no. I'm still trying to figure out my mod loadout and part of that is figuring out what pieces of the USI kit I want to use, but I'll (probably) submit patches for the pieces I'm using.

yalov commented:
thanks
